charts/prometheus-node-exporter/templates/daemonset.yaml

```diff
@@ -205,13 +205,15 @@ spec:
               {{- if .Values.kubeRBACProxy.enableHostPort }}
               hostPort: {{ .Values.service.port }}
               {{- end }}
-            - containerPort: 8888
+            - containerPort: {{ .Values.kubeRBACProxy.healthzPort }}
+              {{- if .Values.kubeRBACProxy.enableHealthzHostPort }}
+              hostPort: {{ .Values.kubeRBACProxy.healthzPort }}
+              {{- end }}
```

Comment:
The proxy opens two ports equal to `service.port` and `proxyEndpointsPort`. The readiness probe must act against the latter as the former requires authorization. 
Naming this same port through another field `healthzPort` seems unnecessary giving an impression of three ports being configured.

The probe's port must be equal to an open port and should not be made independent of it. `healthzPort` should really be `proxyEndpointsPort` - it is the port being and to be probed.

Reply:
Fixed.